src/internal/packager/images/pull_test.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package images provides functions for building and pushing images.
+package images
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/transform"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPull(t *testing.T) {
+	t.Run("pulling a cosign image is successful and doesn't add anything to the cache", func(t *testing.T) {
+
+		ref, err := transform.ParseImageRef("ghcr.io/stefanprodan/podinfo:sha256-57a654ace69ec02ba8973093b6a786faa15640575fbf0dbb603db55aca2ccec8.sig")
+		require.NoError(t, err)
+		tmpDestDir := t.TempDir()
+		tmpCacheDir := t.TempDir()
+		destDir := filepath.Join(tmpDestDir, "images")
+		PullConfig := PullConfig{
```

Comment:
`PullConfig` is a local, unexported variable, so should start with a lowercase character